doc/api/mon_command_api.rst

```diff
@@ -0,0 +1,12 @@
+=====================
```

Comment:
By not having a unique identifier, then it is not possible to reference this section without pointing to a file. Whenever new sections are created that need referencing, add a unique identifier at the very top: 

    .. _unique-target-id:

    =====
    title
    =====